README.markdown

```diff
@@ -908,6 +909,22 @@ UIView.animate(withDuration: 2.0) { [weak self] in
 }
 ```
 
+#### Unused Optional Binding
+
+When the presence of an optional value is required to the context (but not the value itself, i.e., it won't be used), perform a boolean test instead of using optional binding. Prefer `!= nil` over `let _ =` .
+
+**Preferred**:
+```swift
+guard optionalController != nil else { return }
+// do something that requires optionalController but does not use it
+```
```

Comment:
perhaps add an example with an `if` statement?

Reply:
sure thing! don't know why I deleted it (since I had it at first) 🤦🏼